### PR TITLE
NETOBSERV-433 and NETOBSERV-373 - unify names

### PR DESCRIPTION
**Pull request by jotak**
- Use "netobserv" instead of "network-observability" (e.g. as the default namespace)
- Also remove or refresh some outdated doc

Other required PRs:
- operator: https://github.com/netobserv/network-observability-operator/pull/164
- console plugin: https://github.com/netobserv/network-observability-console-plugin/pull/191

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,34 +2,30 @@
...
+    * [flowlogs-pipeline](https://github.com/netobserv/flowlogs-pipeline): A configurable flow collector, enricher and metrics producer.
+    * [netobserv-plugin](https://github.com/netobserv/network-observability-console-plugin):
+    The NetObserv plugin for the Openshift Console.
+* [nflow-generator](https://github.com/netobserv/nflow-generator):
+A fake legacy v5 netflow generator
+* etc.
 
 ### Get started
-The easiest way to get started is to use the [Network Observability Operator](https://github.com/netobserv/network-observability-operator) that will deploy all the components for you.
+The easiest way to get started is to use the [NetObserv Operator](https://github.com/netobserv/network-observability-operator) that will deploy all the components for you. It is available via [OLM](https://operatorhub.io/operator/netobserv-operator).
```
Comment (jpinsonneau) [inline README.md:17]: Do we also plan to rename github repos ?
https://github.com/netobserv/network-observability-operator

Comment (jpinsonneau) [reply README.md:17]: https://github.com/netobserv/network-observability-console-plugin

**Review by jpinsonneau — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jotak) [reply README.md:17]: I fear that would be too much hassle... (and same for quay)

Comment (jpinsonneau) [reply README.md:17]: Seems that github has redirect capabilities for both pages / git commands:
https://docs.github.com/en/repositories/creating-and-managing-repositories/renaming-a-repository

I agree we should not do that if it impacts too much things behind the scene but we may give a try for consistency

**Pull request merged**